detector/tracker/DriftChamber_o1_v02.cpp

```diff
@@ -285,7 +288,9 @@ static dd4hep::Ref_t create_DCH_o1_v02(dd4hep::Detector &desc, dd4hep::xml::Hand
         DCH_length_t cell_rout_zLhalf  = DCH_i->Radius_zLhalf(cell_rout_z0);
         DCH_length_t cell_dz           = DCH_i->Lhalf;
         DCH_angle_t cell_phi_width     = phi_step - safety_phi_interspace;
-        dd4hep::TwistedTube cell_s( cell_twistangle, cell_rin_zLhalf, cell_rout_zLhalf, cell_dz, 1, cell_phi_width);
+        dd4hep::Solid cell_s;
+        if( useG4TT ) cell_s = dd4hep::TwistedTube ( cell_twistangle, cell_rin_zLhalf, cell_rout_zLhalf, cell_dz, 1, cell_phi_width);
+        else          cell_s = CompositeTT(cell_twistangle, cell_rin_z0, cell_rout_z0, cell_dz, cell_phi_width, *DCH_i);
```

Comment:
```suggestion
        dd4hep::Solid cell_s = useG4TT ?
                               dd4hep::TwistedTube(cell_twistangle, cell_rin_zLhalf, cell_rout_zLhalf, cell_dz, 1, cell_phi_width):
                               CompositeTT(cell_twistangle, cell_rin_z0, cell_rout_z0, cell_dz, cell_phi_width, *DCH_i);
```

Reply:
For the sake of readability, I only use a ternary if it fits into one 80-char line, which is not the case